src/emulator/gxm/CMakeLists.txt

```diff
@@ -9,5 +9,5 @@ add_library(
 )
 
 target_include_directories(gxm PUBLIC include)
-target_link_libraries(gxm PUBLIC mem rpcs3 threads vita-headers)
+target_link_libraries(gxm PUBLIC mem glslang rpcs3 shadergen threads vita-headers)
```

Comment:
Should renderer depend on glslang rather than gxm?

Reply:
just removing `glslang` from there worked